### DISCUSSION TITLE
Sort on Search ?

### DISCUSSION THREAD

**Discussion by Amerc0** [TeamNewPipe/NewPipe#11835] [Questions]
### Checklist
 
- [x] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

 ### What is/are your question(s)?

Sorting isn't possible when/after searching. 
Can the simplest sorting (by upload date) be implemented or any PR that has this feature be merged ? 

It is extremely annoying seeing older videos (> 5 yrs) ranking on the top.

**Comment by asandikci** [comment]
See: #8837
